root/release/edit/recordings.tt

```diff
@@ -128,6 +128,12 @@
           [% l('Copy all track artist credits to associated recordings.') %]
         </label>
       </p>
+      <p>
+        [% l('Only check these boxes if the recording titles/artists listed above are incorrect and you want to replace them with the track titles/artists) for this release.') %]
```

Comment:
There's an errant ")" in there, and I feel like a comma would help since the clauses are pretty long
```suggestion
        [% l('Only check these boxes if the recording titles/artists listed above are incorrect, and you want to replace them with the track titles/artists for this release.') %]
```